app/controllers/admin/facilities_controller.rb

```diff
@@ -5,6 +5,7 @@ class Admin::FacilitiesController < AdminController
 
   before_action :set_facility, only: [:show, :edit, :update, :destroy]
   before_action :set_facility_group, only: [:show, :new, :create, :edit, :update, :destroy]
+  before_action :set_available_zones, only: [:new, :create, :edit, :update]
```

Comment:
It might be cleaner to put this behind the feature flag too. I know we _should_ be able to set these zones and not use them (famous last words), but I'm unclear how this'll play out in places we haven't run the backfill yet (eg. Bangladesh)

Reply:
Oops. Yeah, we _should_ feature flag this, since this will ship before the backfill is run anyways (even on IHCI).